src/css/about.css

```diff
@@ -0,0 +1,61 @@
+#header-1 {
+      background-color: rgb(11, 10, 10);
+      font-family: 'Titillium Web';
+      font-size: 56px;
+      color:rgb(246, 243, 242);
+      margin-left: 10%;
+      margin-right: 10%;
+}
+
+#images-box{
+      display: flex;
+      flex-direction: row;
+      justify-content: left;
+      margin-left: 10%;
+      margin-right: 10%;
+}
+
+#image1{
```

Comment:
since #images1, #images2, #images 3 are using the same css styling these can be combined into one
for example
#images1, #images2, #images3 {
 //css styling 
}